docs/adr/0033-content-server-upgrade-to-react.md

```diff
@@ -0,0 +1,140 @@
+
+#### Options
+
+- **Option A: Multiple experiments, no `/beta` route**: Each block of routes will be guarded by separate experiments. If a user is in the experiment, the router will send users to the React version of the page via a paramater. This may not be possible since we cannot access experiment status at the Express level.
+- **Option B: Multiple experiments, `/beta` route**: Each block of routes will be guarded by separate experiments. If a user is in the experiment, the router will send users to the React version of the page via `/beta/<original_url>` with a parameter.
+- **Option C: A single experiment with config settings, and a `/beta` route**: All routes will be guarded by one experiment and separate feature flags per block of routes. In the router file, we will check experiment status as well as feature flag status before send users to the React version of the page via `/beta/<original_url>` with a parameter.
+- **Option D: A single experiment with config settings, and no `/beta` route**: All routes will be guarded by one experiment and separate feature flags per block of routes. In the router file, we will check experiment status as well as feature flag status before send users to the React version of the page via a parameter.
+
+#### Other Considerations
+
+`fxa-content-server` does a lot of things on app startup that Settings does not do, like metrics spin up, recognizing relying parties, OAuth and Sync checks, etc. Because we intend on eventually removing `fxa-content-server`, we need Settings to eventually do these things on app startup instead. When this is implemented, we won't need to access query params provided by content-server.
+
+We may have a follow up ADR to cover the approach to launch `fxa-settings` as the new "base" application depending on complexity and discussions at that time.
+
+### GQL Usage
+
+#### When to Use
+
+We want to extend the GQL functionality that already exists in `fxa-settings` due to the advantages GQL offers (see decision outcomes) and currently, FxA has only implemented GQL on authenticated pages.
+
+- **Option A: Use GQL only on authenticated routes**: Use GQL on authenticated routes, and continue using REST for API calls when users are unauthenticated
+- **Option B: Use GQL everywhere we can**: Add an option per mutation or query to talk to the `graphql-api` without being authenticated, allowing us to use it everywhere **except for likely oauth routes**, which could be another can of worms.
+
+#### Other Implications
+
+If we want to use GQL everywhere we can instead of hitting auth-server directly with REST API calls, and likely even if we're more selective, we have to ensure that the `graphql-api` can handle:
+
+1. Emitting metrics
+2. Sending emails
+3. Sending push notifications (SQS for event-broker)
+
+- **Option I: Create a new API / set of endpoints in `fxa-auth-server` for `graphql-api`**
```

Comment:
Another option is to use the `auth-client` to access auth-server api directly in the graphql server.

Pros - faster to implement, less risk since we already use the auth-client for other routes in graphql
Cons - We will probably have to refactor later since SubPlat architecture will need to send emails, metrics, etc